CHANGES.md

```diff
@@ -1,3 +1,8 @@
+# v0.24.0
+
+* Enhancements:
+  * **Breaking Change** Update cassandra version to 2.1.10 (GH-577)
```

Comment:
I will update CHANGES.md in next release. Basically, the cassandra stress tool is upgraded with the cassandra package. And it is reporting a set of new metrics now and some of the old metrics are also renamed.